index.rst

```diff
@@ -145,16 +154,24 @@ An example report generated by Robot Framework is available `here <_static/tts_c
 
 .. _Robot Framework: https://robotframework.org
 
+One of the key aspects of requirement verification is to be able to map a particular test with a particular requirement it verifies.
+For that, Robot framework allows us to add tags to test reports.
+Using those tags we can assign the list of requirements that are verified by each individual test.
+
 .. _activity-sequence-diagrams:
 
 Activity Sequence Diagrams
 --------------------------
 
+Most of the software requirement verification we perform consist of demonstrating that a particular component executes a particular set of actions.
+These are mostly represented by a set of events, telemetry or event commands published by the component after receiving a particular command or as the by-product of a particular event/telemetry published by another component.
+For example, when a CSC is in the STANDBY state and receives a ``start`` command we expect it to publish a ``summaryState`` event indicating that it transitioned to the DISABLED state.
+
 Activity Sequence Diagrams are broadly used in system design, architecture and systems engineering.
 They allow us to specify _how_ a system works by showing the flow of information between components.
+As such, they provide a way to demonstrate the sequence of communication during a particular activity, which can be used as a verification artifact.
```

Comment:
I guess I don't understand what a "verification artifact" is.
Are the diagrams themselves an artifact themselves? I'd have thought that they would explain some test, and the artifact would be .... whatever... evidence that a CSC published an event... say a chronograf screenshot etc.
Depending on the definition, you may want to rephrase to be:
... which are useful to support and add context to verification artifacts.

Reply:
I think a verification artifact is anything that explain/proof the requirement is met. So, in a sense, the diagrams can be used as an artifact because they both explain the test and provide proof that the requirement is being met, as they are constructed from EFD data...

Anyway, I included your suggested edit...